### DISCUSSION TITLE
ThisIsWin11 1.3.0

### DISCUSSION THREAD

**Discussion by Belim** [builtbybel/ThisIsWin11#173] [Announcements]
- Sign of Life! **The project is alive** 🚀 Had a busy schedule the past couple of months.....I'm slowly waking up again
- [ThisIsWin11] (https://github.com/builtbybel/ThisIsWin11) and [LoveWindowsAgain App](https://github.com/builtbybel/LoveWindowsAgain) have now been merged into one package. 
_The latter application builds on the code of the former and partly vice versa. If you want to do it fast I recommend the LoveWindowsAgain App, for in-depth customizations you can use the ThisIsWin11 monster_

![kT7j5ZxE9A](https://user-images.githubusercontent.com/57478606/185175165-ef120131-5de7-443f-b5bb-bc36f96e63cb.png)

**The LoveWindowsAgain app is only rolled out in the complete TIW11.zip package and is not distributed via the automatic update function.**

_Use the LoveWindowsAgain asset file, to get the standalone version of [LoveWindowsAgain ](https://github.com/builtbybel/LoveWindowsAgain)._
_Please always download the TIW11.zip file as it contains the complete application package._

<hr /><em>This discussion was created from the release <a href='https://github.com/builtbybel/ThisIsWin11/releases/tag/'>ThisIsWin11 1.3.0</a>.</em>

**Comment by DaiichiKuronama** [comment]
How do you restore an app that PumpedApp removed?
